DPGAnalysis/Skims/python/CSCSkim_BFieldStudies_MC_ToscaMap090322_cfg.py

```diff
@@ -27,9 +27,7 @@
 #------------------------------------------
 process.load("Configuration/StandardSequences/Geometry_cff")
 
-process.load("Configuration.StandardSequences.MagneticField_38T_UpdatedMap_cff")
-# trick to make it work with newnew magfield (not in 229)
-process.VolumeBasedMagneticFieldESProducer.version='grid_1103l_090322_3_8t'
+process.load("Configuration.StandardSequences.MagneticField_cff")
```

Comment:
@namapane for my understanding, this replacement is not corresponding to the intended configuration described in the title, so I wonder whether it would not be better just to get rid of this (not the only case)

Reply:
@fabiocos, this set of .pys is quite obviously a relic of CRAFT and it is apparent that they have not been used since. They are also no longer running as far as I can tell. Anyhow the MF configuration file they point to (MagneticField_38T_UpdatedMap_cff) already collapses to the standard sequence, cf:
https://cmssdt.cern.ch/dxr/CMSSW/source/Configuration/StandardSequences/python/MagneticField_38T_UpdatedMap_cff.py
So the inconsistency with the file name is already present, independently of this PR.
Indeed these cfg.py should be considered for removal. As I noted in the PR description, such a cleanup is outside the scope of this PR because I have no elements myself to decide what has to go and what has to be kept maybe for historical reasons. I would prefer such a removal, if deemed appropriate, to happen independently, at least because of the number of signatures involved in approving this PR.